acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/publickeys/KeyIdentifiersAcceptanceTest.java

```diff
@@ -42,20 +41,17 @@
   @ParameterizedTest
   @EnumSource(value = KeyType.class)
   public void noLoadedKeysReturnsEmptyPublicKeyResponse(final KeyType keyType) {
-    initAndStartSigner();
-
+    initAndStartSigner(keyType == BLS ? "eth2" : "eth1");
     validateApiResponse(signer.callApiPublicKeys(keyType), empty());
-    assertThat(signer.walletList()).isEmpty();
   }
 
   @ParameterizedTest
   @EnumSource(value = KeyType.class)
   public void invalidKeysReturnsEmptyPublicKeyResponse(final KeyType keyType) {
     createKeys(keyType, false, privateKeys(keyType));
-    initAndStartSigner();
+    initAndStartSigner(keyType == BLS ? "eth2" : "eth1");
```

Comment:
can we extract the keyType to mode to function that can be reused in these tests

Reply:
Personally, rather carry the Tech debt for now, and get this feature in - fix it if/when it becomes a problem